x-pack/plugins/beats_management/index.ts

```diff
@@ -35,8 +35,8 @@ export function beats(kibana: any) {
     },
     config: () => config,
     configPrefix: CONFIG_PREFIX,
-    init(server: KibanaLegacyServer) {
-      initServerWithKibana(server);
+    async init(server: KibanaLegacyServer) {
```

Comment:
We're changing these to be async for the purpose of error tracking?

Reply:
if they are not async, apparently Kibana ignores them and they get swallowed silently

Reply:
Ok - I am going to test locally. Code LGTM.